web/src/features/AppVersionHistory/AppVersionHistoryRow.jsx

```diff
@@ -281,7 +287,12 @@ class AppVersionHistoryRow extends Component {
                               : newPreflightResults
                               ? "success"
                               : ""
-                          }`}
+                          } 
+                           ${
+                             !showDeployLogs && !showActions
+                               ? "without-btns"
```

Comment:
if showDeployLogs and showActions are false, the two buttons will not be rendered. Therefore, the paragraph needs to transform a bit more.